crates/swc_css_codegen/src/lib.rs

```diff
@@ -312,7 +312,20 @@ where
     fn emit_import_layer_name(&mut self, n: &ImportPreludeLayerName) -> Result {
         match n {
             ImportPreludeLayerName::Ident(n) => emit!(self, n),
-            ImportPreludeLayerName::Function(n) => emit!(self, n),
+            ImportPreludeLayerName::Function(n) if n.value.is_empty() => {
+                // Never emit `layer()`
+                emit!(
+                    self,
+                    swc_css_ast::AtRuleName::Ident(swc_css_ast::Ident {
+                        span: n.span,
+                        value: swc_atoms::js_word!("layer"),
```

Comment:
Please use `js_word` instead

Reply:
![image](https://user-images.githubusercontent.com/15936231/191653312-389161de-d06c-4c29-96c2-e524de684f1a.png)

Reply:
You can import it. I asked to **move** it, not to **remove** it